doc/network_file_desc.rst

```diff
@@ -0,0 +1,196 @@
+:orphan:
+
+.. _network_file_content:
+
+====================
+Network File Content
+====================
```

Comment:
can you add some more information what the file structure is like? what is the file format etc? we need to provide some more information for users if they open the file using other tools and also for documentation